Ask for feedback (#573)
open-telemetry/opentelemetry-dotnet-instrumentation: docs/README.md

[The project board](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/projects/1)
shows the current work in progress.

> ⚠️ **We need you!** ⚠️

Reviewer: I like how this grabs my attention when I first see it.

> Please leave us an **experience report** (in whatever form you like).

Reviewer 2: Leave you feedback
```suggestion
Please, give us your **feedback** (in whatever form you like).
```

You can do this by [submitting a GitHub issue](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/new).

> You may also prefer writting on [Slack](https://cloud-native.slack.com/archives/C01NR1YLSE7).

Reviewer: ```suggestion
You may also prefer writing on [Slack](https://cloud-native.slack.com/archives/C01NR1YLSE7).
```
